### ISSUE TITLE
Staging Review finding: Directions link does not state that it will leave VA.gov

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#111498]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Unified Appointment Experience
**Product:** Appointments
**Feature:** Community Care Direct Scheduling

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

The directions link label does not say it will leave the flow and go to an external site, which can be unexpected.

## Recommended action

Update the link label to "Get directions on Google Maps" or similar to indicate the site is external.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.4 Link Purpose (In Context) - Level A](https://www.w3.org/TR/WCAG21/#link-purpose-in-context)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@StevenStraily @a6marinos this is a duplicate of ADE https://github.com/department-of-veterans-affairs/va.gov-team/issues/111882
